repose-aggregator/components/filters/valkyrie-authorization-filter/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -596,26 +618,32 @@ class ValkyrieAuthorizationFilterTest extends FunSpec with BeforeAndAfterEach wi
     val transformedTenant = "98765"
     val contactId = "123456"
     val filterChain = mock[FilterChain]
-    val mockServletRequest = new MockHttpServletRequest
-    val mockServletResponse = new MockHttpServletResponse
+    var mockServletRequest = new MockHttpServletRequest
```

Comment:
Why would this not be needed?
We need to be able to set the headers on the Request and the wrappers want to close the streams on the Response. Unless the intent was to use the Mokito `when` construct, but that would be painful.